client/containers/DashboardSettings/PubSettings/Doi.js

```diff
@@ -238,6 +257,56 @@ class Doi extends Component {
 		);
 	}
 
+	renderDoi() {
+		const { pubData } = this.props;
+		const { doiSuffix, updating, deleting } = this.state;
+		const fullDoi = this.getFullDoi();
+		const invalidDoi = doiSuffix && !isDoi(fullDoi);
+		const intent = this.getIntent(invalidDoi);
+		const helperText = this.getHelperText(invalidDoi);
+
+		if (this.isDoiEditable()) {
+			return (
+				<FormGroup helperText={helperText} intent={intent}>
+					<ControlGroup>
```

Comment:
This is a matter of taste but I really don't care for the way Buttons look all smushed together in a ControlGroup. I would weakly prefer some space between them.